client/src/components/item/AddOptionDropdown.tsx

```diff
@@ -1,65 +1,78 @@
-import React, {Component} from 'react';
-import {Dropdown} from "semantic-ui-react";
+import React, {Component} from "react";
+import {Dropdown, DropdownProps} from "semantic-ui-react";
 
 interface AddOptionDropdownProps {
-    name: string,
-    required: boolean,
-    placeholder: string,
-    options: Option[]
+    name: string;
+    required: boolean;
+    placeholder: string;
+    options: Option[];
+    loading?: boolean;
+    disabled?: boolean;
+    error?: boolean;
+    value?: string;
+    onChange: (event: React.SyntheticEvent<HTMLElement>, data: DropdownProps, inputName: string) => void;
 }
 
 export interface Option {
-    text: string,
-    value: string
+    text: string;
+    value: string;
 }
 
 interface AddOptionDropdownState {
-    options: Option[],
-    currentValue: string
+    options: Option[];
+    currentValue?: any;
```

Comment:
Why `any` over string? I think it'd be safer to restrict options to strings

Reply:
maybe quantity?

Reply:
Will take a look.  Think the intent was to allow null or something

Reply:
Yeah, turns out I was able to get it to be a string type, just wasn't immediately straightforward